heed/src/mdb/lmdb_ffi.rs

```diff
@@ -1,41 +1,46 @@
 use lmdb_sys as ffi;
 
-pub use ffi::MDB_cursor as MDB_cursor;
-pub use ffi::MDB_dbi as MDB_dbi;
-pub use ffi::MDB_env as MDB_env;
-pub use ffi::mdb_filehandle_t as mdb_filehandle_t;
-pub use ffi::MDB_txn as MDB_txn;
-
-pub use ffi::MDB_APPEND as MDB_APPEND;
-pub use ffi::MDB_CP_COMPACT as MDB_CP_COMPACT;
-pub use ffi::MDB_CREATE as MDB_CREATE;
-pub use ffi::MDB_CURRENT as MDB_CURRENT;
-pub use ffi::MDB_RDONLY as MDB_RDONLY;
-
-pub use ffi::mdb_env_close as mdb_env_close;
+pub use ffi::mdb_filehandle_t;
+pub use ffi::MDB_cursor;
+pub use ffi::MDB_dbi;
+pub use ffi::MDB_env;
+pub use ffi::MDB_envinfo;
```

Comment:
Why this struct has been added?

Reply:
Oops yes missed it with the reformating of everything! It was for another test